pcsx2/gui/AppCoreThread.cpp

```diff
@@ -430,18 +430,21 @@ static void _ApplySettings( const Pcsx2Config& src, Pcsx2Config& fixup )
 	}
 
 	//Till the end of this function, entry CRC will be 00000000
-	if (!gameCRC.Length())
+	bool loadFromDisc = g_Conf->CdvdSource != CDVD_SourceType::NoDisc;
+	if (!gameCRC.Length() && loadFromDisc)
```

Comment:
I suppose we could reuse ```gameCRC.IsEmpty()``` on line 423. That would function the same in this case correct?